### DISCUSSION TITLE
(MacOS) Multiple files dragged into window should yield paths joined by spaces to serve as arguments

### DISCUSSION THREAD

**Discussion by iceman** [ghostty-org/ghostty#4127] [Feature Requests, Ideas] [answered]
This is the expected behavior of OEM terminal

**Comment by jmckillican** [comment]
confirmed that Ghostty does not add space between dropped items while Terminal.app and iTerm do.

**Comment by iceman** [comment]
Companion issue, selecting multiple files, copying and pasting into terminal should give all paths space separated (Terminal and iTerm2).  Currently just the first path pastes.

**Comment by joshdick** [comment]
+1 for this issue. Space-delimiting the names of multiple dragged files makes manipulation as arguments to a prefixed or suffixed command (`cp`, `mv`, etc.) much easier.

**Comment by rorydbain** [comment, marked as answer]
This looks like it's been fixed in https://github.com/ghostty-org/ghostty/pull/4962
